src/platform/pressureFlowSensor.c

```diff
@@ -0,0 +1,93 @@
+#include "stm32f4xx_hal.h"
+#include <platform/pressureFlowSensor.h>
+#include <platform/pressureFlowSensor_private.h>
+#include <platform/configuration_private.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdint.h>
+
+uint32_t read_FS6122_sensor(uint8_t i2c_addr, uint8_t reg_addr){
```

Comment:
Suggest using the scheme defined above where `status_t` is returned and the result is written into a pointer. Like: 

```suggestion
status_t read_FS6122_sensor(uint8_t i2c_addr, uint8_t reg_addr, uint32_t * result){
```